src/random.rs

```diff
@@ -1,7 +1,12 @@
-use rand::{self, Rng};
-use super::Cmd;
+extern crate bytebot_rpn as rpn;
+extern crate rand;
+
+use rand::Rng;
+
+use rpn::Cmd;
+use rpn::Cmd::*;
```

Comment:
Should we `use rpn::Cmd::*;` at the top level, rather than inside the functions that specifically need it? This is an issue with the existing code, so it'd be fine to leave this for later.